learning-bitmark/working-with-bitmarks/creating-bitmark-account.md

```diff
@@ -0,0 +1,122 @@
+# Creating Bitmark account
+
+<br>
+<br>
+## Bitmark account number
+
+Property owners in Bitmark system are identified by their Ed25519 public keys. Those public keys are represented by the Bitmark account numbers which are in based58 endcoded format.
+
+* Account number - livenet example:
+
+    `bqSUHTVRYnrUPBEU48riv9UwDmdRnHm9Mf9LWYuYEa7JKtqgKw`
+
+* Account number - testnet example:
+
+    `fABCJxXc8aYGoj1yLLXmsGdWEo1Y5cZE9Ko5DrHhy4HvgGYMAu`
+
+<br>
+<br>
+## Creating Bitmark account using Bitmark App
+
```

Comment:
Again, I think it's great to ask if additional information could help to clarify a subsection. Something like:

The Bitmark App offers an easy-to-use interface to interact with the Bitmark Property System.

(and similar for the next two sections).